spec/tictactoe_clojure/board_spec.clj

```diff
@@ -1,7 +1,43 @@
 (ns tictactoe-clojure.board-spec
-  (:require [speclj.core :refer :all]
-            [tictactoe-clojure.board :refer :all]))
+	(:require [speclj.core :refer :all]
+		[tictactoe-clojure.board :refer :all]))
+
+(describe "tictactoe-clojure.board"
+	(describe "new-board"
+ 		(it "returns a vector with a range from 0 to 8"
+			(should= new-board [0 1 2 3 4 5 6 7 8] )))
+
+ 	(describe "#valid-slots"
+ 		(it "returns a vector of all the slots for a new board"
+ 			(should= [0 1 2 3 4 5 6 7 8] (valid-slots [0 1 2 3 4 5 6 7 8])))
+ 		(it "returns a vector of the last 8 spots when the first position is taken"
+ 			(should= [1 2 3 4 5 6 7 8] (valid-slots ["X" 1 2 3 4 5 6 7 8])))
+ 		(it "returns a vector of the first row when slots in the 2nd and 3rd row are taken"
+ 			(should= [0 1 2] (valid-slots [0 1 2 "O" "X" "O" "X" "O" "O"]))))
+
+ 	(describe "#valid-move?"
+ 		(it "returns true if the position is availabe in the board"
+			(should= true (valid-move? new-board 1 )))
+ 		(it "returns nil if the position is not availabe in the board"
+			(should= nil (valid-move? [1 2 3 4 5 6 7 8] 0 )))
```

Comment:
`valid-move?` uses the `[some](https://clojuredocs.org/clojure.core/some)` function to determine if the user input is present in the collection of the valid slots. `some` returns nil if the predicate value is not in the collection.